From 8c411c576cec748c8e202ffadf85ad44062c4fbd Mon Sep 17 00:00:00 2001
From: Libin Yang <libin.yang@intel.com>
Date: Wed, 1 Jul 2020 01:50:06 -0400
Subject: [PATCH 34/51] sof-hda-dsp: fixup typo in Hdmi.conf

It should be  "/codecs/hda/hdmi.conf" instead of "/codec/hda/hdmi.conf"

Signed-off-by: Libin Yang <libin.yang@intel.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/sof-hda-dsp/Hdmi.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ucm2/sof-hda-dsp/Hdmi.conf b/ucm2/sof-hda-dsp/Hdmi.conf
index 480e2d6f4203..7b777d1b1d22 100644
--- a/ucm2/sof-hda-dsp/Hdmi.conf
+++ b/ucm2/sof-hda-dsp/Hdmi.conf
@@ -8,7 +8,7 @@ If.hdmi1 {
 			HdmiPCM 3
 			HdmiPrio 500
 		}
-		Include.hdmi1.File "/codec/hda/hdmi.conf"
+		Include.hdmi1.File "/codecs/hda/hdmi.conf"
 	}
 }
 
@@ -20,7 +20,7 @@ If.hdmi2 {
 			HdmiPCM 4
 			HdmiPrio 600
 		}
-		Include.hdmi2.File "/codec/hda/hdmi.conf"
+		Include.hdmi2.File "/codecs/hda/hdmi.conf"
 	}
 }
 
@@ -32,6 +32,6 @@ If.hdmi3 {
 			HdmiPCM 5
 			HdmiPrio 700
 		}
-		Include.hdmi3.File "/codec/hda/hdmi.conf"
+		Include.hdmi3.File "/codecs/hda/hdmi.conf"
 	}
 }
-- 
2.16.4

